assignments/objects.js

```diff
@@ -17,29 +17,51 @@ const example = {
   "gender": "F"
 }
 
+class Intern {
+  constructor (id, email, name, gender){
+  this.id = id;
+  this.name = name;
+  this.email = email;
+  this.gender = gender;
+  }
+  speak() {
```

Comment:
Try this 

```
class Intern {
  constructor (id, email, name, gender){
  this.id = id;
  this.name = name;
  this.email = email;
  this.gender = gender;
  
  this.multiplyNums = function(x,y){
    return (x*y);
  }

  this.speak = function() {return `Hello my name is ${this.name}`};

  }
}
```

It fixes your class methods so that your console.logs for challenge #3 work correctly.

Reply:
Thanks Elvis